### PR TITLE
Add http action to trigger command

### PR DESCRIPTION
**Pull request by heyanlong**

### CODE REVIEW

**Review by kezhenxu94 — COMMENTED** [review]
LGTM, please fix the CI

**Comment by Humbertzhang** [comment]
Some file is named `trigget.go`, i think those should be `trigger.go`, say, `internal/constant/trigget.go` and `internal/flags/trigget.go`

File: commands/trigger/http.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"strings"
+	"time"
+
+	"github.com/apache/skywalking-infra-e2e/internal/flags"
+	"github.com/apache/skywalking-infra-e2e/internal/logger"
+)
+
+type httpAction struct {
+	interval string
+	times    int
```
Comment (fgksgf) [inline commands/trigger/http.go:31]: I think these two attributes can be reused by other actions.

**Comment by fgksgf** [comment]
LGTM

File: commands/trigger/action.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+		i, e := strconv.Atoi(s[:len(s)-1])
+		if e != nil {
+			return time.Second
+		}
+
+		return time.Duration(i) * base
+	}
+	return time.Second
+}
```
Comment (kezhenxu94) [inline commands/trigger/action.go]: Can we just replace these with `time.ParseDuration`?

Comment (heyanlong) [reply commands/trigger/action.go]: So amazing..

**Comment by heyanlong** [comment]
How to fix CI?

**Comment by Humbertzhang** [comment]
The log says 
```
commands/trigger/action.go:30:2: `interval` is unused (structcheck)
	interval time.Duration
	^
commands/trigger/action.go:31:2: `times` is unused (structcheck)
	times    int
	^
```
Maybe You should delete those variables. 
You can try `make lint` in your local to find lint errors.

**Comment by heyanlong** [comment]
```
type httpAction struct {
	action
	url    string
	method string
}
```

Used here, so it cannot be delete.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**